src/js/helpers/Util.js

```diff
@@ -1,6 +1,6 @@
 var consecutiveNumber = 0;
 var objectPath = require("object-path");
-
+var toString = Object.prototype.toString;
```

Comment:
Yeah – gotta say it's a bit obscure how this directly solves the issue mentioned in the PR...